samples/subsys/video/capture/prj.conf

```diff
@@ -4,3 +4,4 @@ CONFIG_SHELL=y
 CONFIG_DEVICE_SHELL=y
 CONFIG_PRINTK=y
 CONFIG_LOG=y
+CONFIG_DISPLAY=y
```

Comment:
Should we enable `CONFIG_DISPLAY` by default in the sample? If we do, we should insure that devices with a display node that is not enabled (IE `status = "disabled"`) still build correctly, as do devices with no display node defined.

Reply:
`CONFIG_DISPLAY` is a generic option just like 'CONFIG_VIDEO'. I tested and see it build with or without `zephyr_display `node defined, so no problem here.

The only thing to care is when `zephyr_display `is defined, its corresponding display node (here eLCDIF) needs to be enabled as well. If not, it will not build (despite of `CONFIG_DISPLAY` enabled or not)